Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EndOfYearDataManager.swift

```diff
@@ -362,6 +392,21 @@ public struct ListenedNumbers {
     }
 }
 
+public struct YearOverYearListeningTime {
+    public let totalPlayedTimeThisYear: Double
+    public let totalPlayedTimeLastYear: Double
+
+    public var percentage: Double {
+        let nonRoundendPercentage = ((100 * totalPlayedTimeThisYear) / totalPlayedTimeLastYear) - 100
+        return (nonRoundendPercentage.rounded() * 100) / 100
```

Comment:
This looks like we're calculating the percent change between this and last years listening number. 

Wouldn't it be the same to do:

```suggestion
        (((totalPlayedTimeThisYear - totalPlayedTimeLastYear) / totalPlayedTimeLastYear) * 100).rounded()
```

Reply:
Also since we're using this in 4 different places in the view, and the values aren't going to change could this to be calculated in init and stored?

Reply:
Done in b344100174bf440aeb2f3e6d27192b0574a44e84